File: ny_telemedicine/lib/Models/schedule.dart
class Schedule {
  int? id;
  String? email;
  String? date;
  String? startTime;
  String? duration;
  String? patientName;

  Schedule({this.id, this.email, this.date, this.startTime, this.duration, this.patientName});

  // Schedule.fromJson(Map<String, dynamic> json) {
  //   email = json['email'];
  //   date = json['date'];
  //   startTime = json['startTime'];
  //   duration = json['duration'];
  // }

  factory Schedule.fromJson(Map<String, dynamic> json) {
    return Schedule(
        id: json['id'],
        email : json['email'],
        date : json['date'],
        startTime : json['startTime'],
        duration : json['duration'],
        patientName : json['patientName']
    );
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['email'] = this.email;
    data['date'] = this.date;
    data['startTime'] = this.startTime;
    data['duration'] = this.duration;
    data['patientName']=this.patientName;
    return data;
  }
}

File: BackEnd/AccountService/Account/src/main/java/com/example/account/Model/Schedule.java
package com.example.account.Model;

import javax.persistence.*;

@Entity
public class Schedule {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Integer id;

    @Column(name = "email")
    private String email;

    @Column(name = "date")
    private String date;

    @Column(name = "startTime")
    private String startTime;

    @Column(name = "duration")
    private String duration;

    @Column(name = "patientName")
    private String patientName;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getStartTime() {
        return startTime;
    }

    public void setStartTime(String startTime) {
        this.startTime = startTime;
    }

    public String getDuration() {
        return duration;
    }

    public void setDuration(String duration) {
        this.duration = duration;
    }


    public String getpatientName() {
        return patientName;
    }

    public void setpatientName(String patientName) {
        this.patientName = patientName;
    }
}
